SenderEmail/src/main/java/com/example/SenderEmail/exception/ErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.example.SenderEmail.exception;
+
+public class ErrorResponse {
+    private int statusCode ;
+    private  String  message ;
```

Comment:
Always use `final` for fields to prevent unexpected changes and errors.

Reply:
done